e2e/test-backup.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+load "lib/utils"
+load "lib/detik"
+load "lib/k8up"
+
+# shellcheck disable=SC2034
+DETIK_CLIENT_NAME="kubectl"
+# shellcheck disable=SC2034
+DETIK_CLIENT_NAMESPACE="k8up-e2e-subject"
+# shellcheck disable=SC2034
+DEBUG_DETIK="true"
+
+@test "verify a backup" {
+	given_running_operator
+	given_s3_storage
+	given_a_subject
+
+	apply definitions/backup
+	try "at most 10 times every 1s to get backup named 'k8up-k8up-test' and verify that '.status.started' is 'true'"
+	try "at most 10 times every 1s to get job named 'k8up-k8up-test' and verify that '.status.active' is '1'"
+	try "at most 10 times every 2s to find 1 pod named 'k8up-k8up-test-.....' with 'status' being 'Running'"
+
+	# Wait for completion
+	try "at most 10 times every 2s to get backup named 'k8up-k8up-test' and verify that '.status.finished' is 'true'"
```

Comment:
I would check for a condition. The "finished" flag is kind of deprecated.
you could also use this:
`kubectl -n $DETIK_CLIENT_NAMESPACE wait --for condition=ready backup/k8up-k8up-test --timeout 1m`
But I'm fine with the `try` command

Reply:
I guess you meant condition=completed?